pbs_light.go

```diff
@@ -91,6 +92,15 @@ type bidResult struct {
 	bid_list pbs.PBSBidSlice
 }
 
+const DEFAULT_PRICE_GRANULARITY = "med"
```

Comment:
the account object on the site updates postgres, which is read by the cache. so if you update the cache, you'll be ready for the actual changes to the site.